providers/tests/fab/auth_manager/api_endpoints/test_backfill_endpoint.py

```diff
@@ -188,7 +188,7 @@ class TestCreateBackfill(TestBackfillEndpoint):
     def test_create_backfill(self, session, dag_maker):
         with dag_maker(session=session, dag_id="TEST_DAG_1", schedule="0 * * * *") as dag:
             EmptyOperator(task_id="mytask")
-        session.add(SerializedDagModel(dag))
+        SerializedDagModel.write_dag(dag)
```

Comment:
```suggestion
        SerializedDagModel.write_dag(dag, session=session)
```

Probably should use the same session?

Reply:
I used it a lot without the session in the test. Does it matter, if yes, happy to change it in all places but I think it does not matter in tests

Reply:
Probably doesn't really matter. This one just stood out in the diff because it was `session.add` before.